wp-content/mu-plugins/the-world-site-config/configs/global/global-plugins.php

```diff
@@ -18,6 +18,7 @@
 		'microsoft-start/index.php',
 		'pwa/pwa.php',
 		'redirection/redirection.php',
+		'redis-cache/redis-cache.php',
```

Comment:
FYI this is where plugins get activated.

Reply:
Not sure if this needs to be activated in dev env. If this is meant to replace the pantheon page cache plugin, it was only in the production plugins.